keel-web/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -52,6 +52,7 @@ class DefaultConfiguration {
 
   @Bean
   @Primary
+  @Qualifier("jsonMapper")
```

Comment:
why was `@Qualifier("jsonMapper")` added?

Reply:
Because I rather name the object `jsonMapper`, and since the object created by default configuration comes from a function called `objectMapper`, Spring can't find it automatically unless I use a qualifier. I prefer `jsonMapper` to contrast it with the `yamlMapper` used in some of the classes (both of which are instances of `ObjectMapper`, so I prefer to avoid confusion).